package.json

```diff
@@ -29,7 +29,8 @@
     "node": ">=14.19.1"
   },
   "optionalDependencies": {
-    "openmct": "^2.0.5"
+    "openmct": "^2.0.5",
+    "file-saver": "^2.0.5"
```

Comment:
I think we make this a dev dependency. Also, let's lock version, we are using dependabot to manage our dependency versions.